keyboards/handwired/dactyl_manuform_rp2040/keymaps/default/keymap.c

```diff
@@ -0,0 +1,73 @@
+                                  KC_TRNS, TO(_QWERTY),             KC_TRNS, KC_TRNS),
+
+	[_ARROWS] = LAYOUT(
+    KC_F12, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5,                              KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11,
+    KC_NUM, KC_TRNS, KC_LPRN, KC_RPRN, KC_TRNS, KC_TRNS,                    LCTL(KC_LEFT), LCTL(KC_DOWN), LCTL(KC_UP), LCTL(KC_RGHT), KC_CAPS, KC_NUM,
+    KC_MINS, KC_EQL, KC_LBRC, KC_RBRC, KC_BSLS, KC_GRV,                     KC_LEFT, KC_DOWN, KC_UP, KC_RGHT, KC_END, KC_SCRL,
+    KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE, KC_TILD,                   KC_HOME, KC_MPLY, KC_MPRV, KC_MNXT, KC_MUTE, KC_INS,
+                                  KC_TRNS, KC_TRNS,                 KC_VOLD, KC_VOLU,
+                                  LALT(KC_SPC), LGUI(KC_ENT),       TO(_QWERTY), KC_TRNS,
+                                  KC_TRNS, KC_TRNS,                 KC_TRNS, KC_TRNS,
+                                  KC_TRNS, KC_TRNS,                 KC_TRNS, KC_TRNS),
+
+	[_MOUSE] = LAYOUT(
+    KC_ACL0, KC_NO, KC_BTN4, KC_BTN3, KC_BTN5, KC_NO,                       KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, QK_BOOT,
+    KC_ACL1, KC_NO, KC_BTN1, KC_MS_U, KC_BTN2, KC_NO,                       KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_ACL2, KC_NO, KC_MS_L, KC_MS_D, KC_MS_R, KC_NO,                       KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_NO, KC_NO, KC_WH_L, KC_WH_U, KC_WH_D, KC_WH_R,                       KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+                                  KC_TRNS, KC_NO,                   KC_TRNS, KC_TRNS,
+                                  KC_TRNS, KC_TRNS,                 KC_TRNS, KC_TRNS,
+                                  KC_TRNS, KC_TRNS,                 KC_TRNS, KC_TRNS,
+                                  KC_TRNS, TO(_QWERTY),             KC_TRNS, TO(_QWERTY)),
+
+	[_GAMING] = LAYOUT(
+    KC_6, KC_1, KC_2, KC_3, KC_4, KC_5,                                     KC_HOME, KC_NUM, KC_PSLS, KC_PAST, KC_PMNS, KC_LPRN,
+    KC_7, KC_TAB, KC_Q, KC_W, KC_E, KC_R,                                   KC_END, KC_P7, KC_P8, KC_P9, KC_PPLS, KC_RPRN,
+    KC_ESC, KC_LSFT, KC_A, KC_S, KC_D, KC_F,                                KC_LT, KC_P4, KC_P5, KC_P6, KC_PENT, KC_PEQL,
+    KC_8, KC_LCTL, KC_Z, KC_X, KC_C, KC_V,                                  KC_GT, KC_P1, KC_P2, KC_P3, KC_PCMM, KC_CALC,
+                                  KC_9, KC_0,                       KC_P0, KC_PDOT,
+                                  KC_SPC, KC_ENT,                   KC_TRNS, KC_TRNS,
+                                  KC_G, KC_M,                       KC_TRNS, KC_TRNS,
+                                  KC_LALT, TO(_QWERTY),             KC_TRNS, TO(_QWERTY))
+};
+
+#if defined(ENCODER_ENABLE) && defined(ENCODER_MAP_ENABLE)
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][NUM_DIRECTIONS] = {
+
+};
+#endif // defined(ENCODER_ENABLE) && defined(ENCODER_MAP_ENABLE)
+
+
+
+
+
```

Comment:
code block not required as encoders not set to true and the code block is essentially null/functionless
```suggestion
```